aspnetcore/security/blazor/webassembly/hosted-with-azure-active-directory-b2c.md

```diff
@@ -230,8 +226,6 @@ If adding authentication to an app, manually add the package to the app's projec
     Version="{VERSION}" />
```

Comment:
Should we just replace `{VERSION}` with 3.2.0?

Reply:
Uh ... this future-proofs the topic for future versions tho. I'll do it, but it creates work later, and we don't have the best track record with hard-coded versions around here. 🙈

Reply:
Is there a way that we could use a version placeholder that gets centrally configured and replaced as part of the build?

Reply:
The INCLUDE would encompass the code block. It's kind'a pretty much not worth the effort to have a separate file. When changing one, I just do a find-'n-replace on the string to lick them all in 22 nanoseconds.

Since we'll be using *whole-topic versioning* at 5.0 and have whole new copies of each doc at 5.0, it won't really matter unless there's a 3.3 (... and 3.4 ... and so on).